angular-app/src/app/discovery/discovery.component.css

```diff
@@ -0,0 +1,54 @@
+/* Add application styles & imports to this file! */
+@import '~@angular/material/prebuilt-themes/deeppurple-amber.css';
+
+mat-chip {
+  max-width: 200px;
+}
+
+.pi-ui .pi-chip-filter .pi-chip-filter-dropdown {
+	 cursor: default;
+}
+ .mat-menu-panel.pi-chip-filter-menu {
```

Comment:
Is there an extra beginning space on this line and the other `.mat-*` lines below? If so, please remove and verify everything aligns.